### PR TITLE
python@3.9: bump setuptools & pip resources

### PR DESCRIPTION
**Pull request by unitof**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Bump outdated resources `setuptools` and `pip`

### CODE REVIEW

**Comment by unitof** [comment]
Recommended by @carlocab https://github.com/Homebrew/homebrew-core/pull/107080/#issuecomment-1201904354

File: Formula/python@3.9.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class PythonAT39 < Formula
   url "https://www.python.org/ftp/python/3.9.13/Python-3.9.13.tar.xz"
   sha256 "125b0c598f1e15d2aa65406e83f792df7d171cdf38c16803b149994316a3080f"
   license "Python-2.0"
-  revision 1
+  revision 3
```
Comment (carlocab) [inline Formula/python@3.9.rb:7]:
```suggestion
  revision 2
```

**Review by carlocab — COMMENTED** [review]
Thanks. Could you squash all your commits together and use the commit message
```
python@3.9: update resources
```
?

**Comment by unitof** [comment]
Could use some advice on what needs to be done.

All macOS Intels timed out.

On 11 and 12 arm64: `chapel` & `protobuf@3` fail tests.

On linux, 21 formulae fail tests.

**Comment by carlocab** [comment]
We need a long build slot to avoid the timeout. We'll just have to wait for one to free up.

Can you look at the `chapel` and `protobuf@3` error log to see if it might be related to the changes here?

Most of the Linux failures might be fixed by a rebase.

**Comment by unitof** [comment]
Sounds good!

Does it automatically run when a long slot opens up, or is that something y'all monitor and rerun manually?

Will see what I can do about `chapel` and `protobuf@3`.

**Comment by carlocab** [comment]
Right now this is monitored manually, but we're working on automating it.

**Review by cho-m — APPROVED** [review]

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**